phpcs.xml

```diff
@@ -11,6 +11,9 @@
 		</properties>
 	</rule>
 
+	<rule ref="WordPress.Files.FileName.InvalidClassFileName">
+		<exclude-pattern>tests/*</exclude-pattern>
+	</rule>
```

Comment:
Almost all of the test files begin with `test-`, while this rule states that they should begin with `class-`.